### DISCUSSION TITLE
[Feature] sort by date in smart search

### DISCUSSION THREAD

**Discussion by ilyaChuk** [immich-app/immich#11520] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I guess now the images in the search are sorted by the percentage of match. but I still sometimes want to see newer photos at the top of the list, not super old ones. maybe we can sort by date with a given minimum percentage of match?

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Kvaks** [comment]
Yes! I just assumed this was already a feature and I just couldn't find it in the UI. I almost ALWAYS want my results sorted by reverse chronological order. If I smart-search for, say,  "screenshot", I don't need it sorted by which screenshot is more screenshot than the other screenshots, I want the latest first. Same if I search for "bicycle" - they are all bicycles, I don't typically want the most "bicycle-ish" first, but the newest images of bicycles first!
